tests/requirements/requirements_base.txt

```diff
@@ -10,7 +10,9 @@ mock
 pillow
 pyflakes>=2.1.1
 python-dateutil
+mysqlclient==2.0.3
+psycopg2
 
 # Unreleased django-cms 4.0 compatible packages
-https://github.com/django-cms/django-cms/tarball/develop-4#egg=django-cms
-https://github.com/divio/djangocms-text-ckeditor/tarball/support/4.0.x#egg=djangocms-text-ckeditor
+git+https://github.com/django-cms/django-cms@develop-4#egg=django-cms
```

Comment:
It's fine to change these, they were in the previous format because the Divio Cloud didn't like Pip urls, it's was more convenient to use the same urls across the projects i..e in and out of the Divio Cloud.

Reply:
Was not aware of this. Will revert.